Formula 1/ViewModels/CircuitsViewModel.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  CircuitsViewModel.swift
+//  Formula 1
+//
+//  Created by Gio on 11/13/19.
+//  Copyright © 2019 Gio. All rights reserved.
+//
+
+import Foundation
+
+class CircuitsViewModel: ObservableObject {
+    @Published private var circuits = [Circuit]()
```

Comment:
Why do you make this a @Published but then use a delegate? Aren't these two patterns in conflict? You're effectively not using the @Published at all.

Reply:
This is for it to work in SwiftUI. I haven't decided on the best way to set up both ways in a single class

Reply:
oh I see. That being the case document it. Documentation should explain the why. Future you and future teammates will thank you for it.

Also, consider using a closure instead of a delegate.
`var onDataChanged: ([Circuit]) -> Void = { _ in }`

Reply:
Please address the comment before collapsing conversation.

Reply:
I removed @Published wrapper